celerdata/README.md

```diff
@@ -0,0 +1,67 @@
+# Agent Check: celerdata
+
+## Overview
+
+[CelerData][1] is the only SQL engine that is fast enough to run the most demanding workloads directly on your data lakehouse, all powered by StarRocks and cloud-managed by CelerData.
```

Comment:
For integrations-extras, the overview should avoid marketing material. It should include: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful). Potential ew customers should be able to understand the integration 
	3	What types of data sent and the value

Reply:
done